main.py

```diff
@@ -834,7 +788,7 @@ def StartPlaygroundGame(self):
         # Send POST request
         response = self.HttpRequest(url, headers, "POST", 200)
 
-        if response is None:
+        if response is None or not response:
```

Comment:
According to this commit: https://github.com/masterking32/MasterHamsterKombatBot/commit/820f0b7d938b62578e4bdc597ebe4bd78464e54f , these changes are not needed anymore!